Zend/tests/type_declarations/typed_class_constants_002.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+Typed class constants (type mismatch; compile-time)
+--FILE--
+<?php
+class A {
+	public const string A = 1;
+}
+?>
+--EXPECTF--
+Fatal error: Cannot use int as value for class constant A::A of type string in %s on line %d
```

Comment:
It would probably be good to test the reported line too.

Reply:
@iluuu1994 sure, will do.

Reply:
After looking at other tests, I'm not convinced that this is needed. I haven't touched anything error reporting related and modifying tests becomes a hassle with explicit line numbers.